src/spikeinterface/preprocessing/unsigned_to_signed.py

```diff
@@ -15,17 +15,18 @@ class UnsignedToSignedRecording(BasePreprocessor):
     def __init__(
         self,
         recording,
+        bit_depth=None,
```

Comment:
Can you add docstrings as well below the class definition to explain this new behavior?